toolkit/tools/liveinstaller/liveinstaller.go

```diff
@@ -190,7 +190,13 @@ func removeOldAzureLinuxBootTargets() (err error) {
 
 func ejectDisk() (err error) {
 	logger.Log.Info("Ejecting CD-ROM.")
-	_, _, err = shell.Execute("eject", "--cdrom")
+	const squashErrors = false
+	program := "eject"
+	commandArgs := []string{
+		"--cdrom",
+		"--force",
```

Comment:
is this comma necessary?

Reply:
yes, I was getting a go syntax error about "no comma before newline in composite syntax" without it being present.